accepted/2021/nano-framework-tfm/nan-framework-tfm.md

```diff
@@ -0,0 +1,61 @@
+# Nano Framework TFM
+
+**Owner** [Immo Landwerth](https://github.com/terrajobst)
+**Owner** [Laurent Ellerbach](https://github.com/Ellerbach)
+**Owner** [José Simões](https://github.com/josesimoes)
+
+While .NET has been optimized to be a general purpose application development
+framework, we also created a flavor of .NET that can be used in the context of
+very resource-constrained hardware, called [.NET Micro Framework][netmf]. To
+give you an idea what resource-constrained looks like: 256 KB of flash and 64 KB
+of RAM. As such, the .NET Micro Framework doesn't have many features of .NET,
+such as generics.
+
+While the [.NET Micro Framework][netmf] is archived, there is a community
+version of it called the [.NET Nano Framework][nanoframework].
+
+Because of the resource constraints there isn't much of a point of being able to
+share binaries between .NET Nano Framework and the rest of the .NET ecosystem.
+Also, it simply wouldn't even be feasible for it to implement .NET Standard.
+Thus, there are no plans to make .NET Micro Framework compatible with .NET
+Standard or the new [.NET 5 based TFMs][net5-tfm].
+
+However, in order for the .NET Micro Framework to have a tooling experience in
```

Comment:
I guess for this one, you ment .NET nanoFramework rather than netmf?

Reply:
Yep :-)